apps/api/src/app/bridge/usecases/sync/sync.usecase.ts

```diff
@@ -74,13 +74,13 @@ export class Sync {
       });
     }
 
-    const createdWorkflows = await this.createWorkflows(command, discover.workflows);
+    const persistedWorkflowsInBridge = await this.createWorkflows(command, discover.workflows);
```

Comment:
I am not sure if the term makes sense as it is persisted both in the bridge and novu database, in this context it is related to the novu database.

Reply:
persisted reffers to the database